roles/distributive/tasks/main.yml

```diff
@@ -15,7 +15,7 @@
 - name: install distributive, health check packages for current role
   become: yes
   yum:
-    state: present
+    state: latest
```

Comment:
if we do `latest`, though, aren't we opening ourselves up for problems down the road if we release a new version of distributive that is incompatible with this release? this is what happened with mantl-dns in #1265. Can we just explicitly tell it to install `0.2.5-1`?